src/common/libflux/handle.c

```diff
@@ -278,6 +285,8 @@ flux_t *flux_open (const char *uri, int flags)
                           | FLUX_O_TEST_NOSUB
                           | FLUX_O_RPCTRACK;
 
+    err_init (errp);
+
```

Comment:
Are we going to make callers do something like`error.text[0] ? error.text : strerror (errno)` to decode errors?

Seems like the `flux_error_t` should always be filled when provided and there is an error.

Reply:
Good point. Sorry that was short sighted. I'll fix it if possible